src/Container.ts

```diff
@@ -72,7 +72,7 @@ export default class Container {
       config.facilitator.chains[configAuxChainId].subGraphRpc,
     );
 
-    const services = Services.create(repositories, config);
+    const services = Services.create(repositories, config, gatewayAddresses);
```

Comment:
mosaicConfig is wrapped under config object. we can replace that with gatewayAddresses. 
This will be clean. Everywhere  then we can call GatewayAddresses method.

Reply:
Done